flask_resty/api.py

```diff
@@ -19,6 +20,10 @@ def handle_api_error(error):
 
 
 def handle_http_exception(error):
+    if isinstance(error, RoutingException):
+        # This is not actually an error. Forward it to properly redirect.
+        return error
```

Comment:
returning is the right way to get it to continue ?

Reply:
Yes.